src/AzureBlobStorageServiceProvider.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+namespace A17\Twill;
+
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\ServiceProvider;
+use League\Flysystem\AzureBlobStorage\AzureBlobStorageAdapter;
+use League\Flysystem\Filesystem;
+use MicrosoftAzure\Storage\Blob\BlobRestProxy;
+
+class AzureBlobStorageServiceProvider extends ServiceProvider
+{
+    /**
+     * Perform post-registration booting of services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        Storage::extend('azure', function ($app, $config) {
+            $connectionString = sprintf(
+                'DefaultEndpointsProtocol=%s;AccountName=%s;AccountKey=%s;EndpointSuffix=%s',
+                isset($config['use_https']) ? 'http' : 'https',
```

Comment:
This doesn't sound right anymore.
```suggestion
                ($config['use_https] ?? true) ? 'https' : 'http',
```